src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -309,6 +309,13 @@ public void FullName_CurrentlyExecutingAssembly()
             Assert.Equal(assemblyName.Name.Length, assemblyName.FullName.IndexOf(','));
         }
 
+        [Fact]
+        public void EmptyFusionLog()
+        {
+            FileNotFoundException fnfe = Assert.Throws<FileNotFoundException>(() => Assembly.LoadFrom(@"c:\non\existent\file.dll"));
```

Comment:
Nit: Strange path to use on Unix.

Reply:
It's particularly likely to be nonexistent on Unix 😺

Reply:
Sure. But if we're going to use something that's a file name rather than a path, send better to use something that's a file name on both OSes.